azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -85,12 +85,30 @@ stages:
         }
       name: echovar
       displayName: Echo branch name
+  # Check PR Label
```

Comment:
suggestion:
We can make this a template, which enumerates all the PR labels and stores them in an output variable.
That way we can extend this functionality for other purposes if needed.

Reply:
Oh, that's a useful suggestion!